src/tests/080-commentTest.js

```diff
@@ -0,0 +1,134 @@
+import chai from 'chai';
+import app from '../index';
+import mockData from './mock/mockData';
+import tripMockData from './mock/tripMockData';
+
+const { expect } = chai;
+const { oneWaytrip } = tripMockData;
+const {
+  userComment, user1, user2, newSuperAdminLogin
+} = mockData;
```

Comment:
Probably other teammates will use this file. In order to avoid complexity, wrap mockData into an object. use it to point to property. 
example: `mockData.userComment `